### ISSUE TITLE
Create Primary Template View

### ISSUE THREAD

**Issue by timothyjamesjensen** [new-dev/Tabrir#3]
The goal of this is to create the primary template view. It should have the following features
- NOW
  - New Group button that takes user to new group view
- Future
  - If there are no stored groups, display a message welcoming a new user and prompting them to create their first url group
  - iterate over stored groups and display them as clickables

You can close this story once the now portion is done if you want, thats all we really care about at the moment. We can make another story for the future features.
